### ISSUE TITLE
Error when using --backtrace flag and config file

### ISSUE THREAD

**Issue by DominikGuzei** [xolvio/chimp#444]
#####  Expected behaviour
Chimp should run like normal and provide better backtraces through the (hidden?) option `--backtrace`

#####  Actual behaviour
Chimp failes immediately with the following error:
```
shell:code user$ ./bin/test/acceptance_run
/path/to/code/project/node_modules/.bin/chimp /path/to/code/config/chimp_dev.js  --backtrace

[chimp] Running...
fs.js:887
  return binding.lstat(pathModule._makeLong(path));
                 ^

Error: ENOENT: no such file or directory, lstat '/path/to/code/true'
    at Error (native)
    at Object.fs.lstatSync (fs.js:887:18)
    at Object.realpathSync (fs.js:1518:21)
    at Object.expandPathWithExtensions (/path/to/code/project/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/path_expander.js:14:23)
    at /path/to/code/project/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/path_expander.js:8:27
    at Array.map (native)
    at Object.expandPathsWithExtensions (/path/to/code/project/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/path_expander.js:7:31)
    at Object.expandPaths (/path/to/code/project/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/feature_path_expander.js:9:38)
    at Function.Configuration (/path/to/code/project/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/configuration.js:21:63)
    at getConfiguration (/path/to/code/project/node_modules/chimp/dist/lib/cucumberjs/cucumber-wrapper.js:47:38)
Cucumber steps failed
Chimp Watch exited with code 2
```

##### Steps to reproduce the problem
- Run chimp with a config file + `--backtrace` flag

#####  Used version

* Chimp: 0.38.0
* Node.js: 4.4.7
* Operation system: MacOS 10.9.5

#####  Config File
```
const path = require('path');
// Defaults: https://github.com/xolvio/chimp/blob/master/src/bin/default.js
module.exports = {
  path: path.resolve('code/project/test/acceptance/tests'),
  serverPort: 3100,
  serverHost: 'localhost',
  chai: true,
  ddp: 'http://localhost:3100',
  webdriverio: {
    waitforTimeout: 10000
  },
  mochaTimeout: 10000
};
```
------- 
*Join our Slack [xolv.io/community](http://xolv.io/community) #chimp channel, where you can find help and help others.*

**Comment by samhatoum** [comment]
Sorry this is taking so long @DominikGuzei, you know how life gets :)

Does this happen if you add `--backtrace=false` ?

**Comment by DominikGuzei** [comment]
Hey @samhatoum no worries, it's not mission critical but just does not work correctly 😉 
Running it with false yields exactly the same error (but with `false`)

```
=> App running at: http://localhost:3100/

[chimp] Running...
fs.js:839
  return binding.lstat(pathModule._makeLong(path));
                 ^

Error: ENOENT: no such file or directory, lstat '/Users/dominik/work/projects/input-output/sales-app/code/false'
    at Error (native)
    at Object.fs.lstatSync (fs.js:839:18)
    at Object.realpathSync (fs.js:1439:21)
    at Object.expandPathWithExtensions (/Users/dominik/work/projects/input-output/sales-app/code/attainenroll/node_modules/cucumber/lib/cucumber/cli/path_expander.js:14:23)
    at /Users/dominik/work/projects/input-output/sales-app/code/attainenroll/node_modules/cucumber/lib/cucumber/cli/path_expander.js:8:27
    at Array.map (native)
    at Object.expandPathsWithExtensions (/Users/dominik/work/projects/input-output/sales-app/code/attainenroll/node_modules/cucumber/lib/cucumber/cli/path_expander.js:7:31)
    at Object.expandPaths (/Users/dominik/work/projects/input-output/sales-app/code/attainenroll/node_modules/cucumber/lib/cucumber/cli/feature_path_expander.js:9:38)
    at Function.Configuration (/Users/dominik/work/projects/input-output/sales-app/code/attainenroll/node_modules/cucumber/lib/cucumber/cli/configuration.js:21:63)
    at getConfiguration (/Users/dominik/work/projects/input-output/sales-app/code/attainenroll/node_modules/chimp/dist/lib/cucumberjs/cucumber-wrapper.js:47:38)
Cucumber steps failed
Chimp Watch exited with code 2
```

**Comment by samhatoum** [comment]
Thanks @DominikGuzei 

@lgandecki this might be as simple as [adding it here](https://github.com/xolvio/chimp/blob/master/src/bin/chimp.js#L20), but I've not looked deeper

**Comment by lucetius** [comment]
Merged with #461
